voxels.def

```diff
@@ -328,4 +328,6 @@ voxel "voxels/3657_VALVE2_G.kvx"		{ tile 3657 }
 voxel "voxels/3658_VALVE2_H.kvx"		{ tile 3658 }
 
 voxel "voxels/3659_SOCKET1.kvx"			{ tile 3659 }
-voxel "voxels/3660_SOCKET2.kvx"			{ tile 3660 }
\ No newline at end of file
+voxel "voxels/3660_SOCKET2.kvx"			{ tile 3660 }
+
+include maphacks.def
```

Comment:
I would add `**include**` on top of everything here.

Reply:
moved include on top. Wasn't sure if voxels had to be loaded prior